app/helpers/validation_helper.rb

```diff
@@ -43,7 +43,7 @@ def error_messages_for(object)
   
   def error_messages_formatted(errors, intro = "")
     return unless errors && !errors.empty?
-    error_messages = errors.map { |msg| content_tag(:li, msg.gsub(/^(.*)\^/, '').html_safe) }.join("\n").html_safe
+    error_messages = errors.map { |msg| content_tag(:li, msg.gsub(/^(.*?)\^/, "").html_safe) }.join("\n").html_safe
```

Comment:
Can you add a test for this error message? It should be similar to:

https://github.com/otwcode/otwarchive/blob/2d92ec1e100d2db8d98c791258b8ea115044975d/features/works/work_edit_tags.feature#L69-L76